src/platform/mtk/dai.c

```diff
@@ -0,0 +1,292 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright(c) 2024 Google LLC.  All rights reserved.
+// Author: Andy Ross <andyross@google.com>
+#include <sof/lib/dai-legacy.h>
+#include <ipc/dai.h>
+#include <sof/drivers/afe-drv.h>
```

Comment:
this isn't yet replacing src/platform/mt81*/lib/dai.c, right? But eventually it will?

Reply:
It **is** replacing the platform layers.  It's still using the legacy driver code in src/drivers/mediatek/afe, but configuring it from DTS.